overrides/main.html

```diff
@@ -1,12 +0,0 @@
-{#-
```

Comment:
Yeah, lets get rid of this. Maybe later we could introduce a similar hint for the v2 version ...

Reply:
ack